pages/04.tutorials/32.confiugre-forbidden-sysctls/default.en.md

```diff
@@ -0,0 +1,163 @@
+---
+title: 'How to reconfigure forbidden sysctls'
```

Comment:
Better call them `unsafe` sysctls instead of `forbidden`.

Reply:
fixed